packages/vrm-viewer/src/features/vrm-mixin.ts

```diff
@@ -1,6 +1,11 @@
 import { VRM, VRMLoaderPlugin, VRMUtils } from '@pixiv/three-vrm';
+import { createVRMAnimationClip, VRMAnimationLoaderPlugin, VRMLookAtQuaternionProxy } from '@pixiv/three-vrm-animation';
```

Comment:
I think it's rare to display models as T-pose so it's ok to always  import this.